### PR TITLE
Apply context bar HTML into template

### PR DESCRIPTION
**Pull request by LornaHarwood**
## Changes in this PR

The Context bar HTML, css and functionality has been added to the `publication.html` template.

This is part of #164 

## Screenshots of UI changes

Screenshot 2022-04-25 at 12 21 43

## Next steps

The Download button does not have any functionality yet as there is nowhere to send this yet. Once the PDF Download feature has been implemented, this can be linked up with the frontend.

### CODE REVIEW

File: cms/publications/templates/publications/publication.html [hunk 1]
```diff
@@ -2,16 +2,79 @@
...
+        </div>
+        <!-- <div class="nhsuk-body-s">
+        {{ self.first_published_at|date }} | Author: {{ self.author }}
+        </div> -->
+        <div class="nhsuk-u-reading-width">
+        <div class="nhsuk-body-s">
+            {% for category in self.categorypage_category_relationship.all %}
+            <a href="{{ self.get_parent.url }}?category={{ category.category.id }}">{{ category.category }}</a>
+            {% endfor %} <br>
+            <strong>Content type:</strong> Publication (guidance)
```
Comment (jacksonj04) [inline cms/publications/templates/publications/publication.html:21]: The `Publication (guidance)` here is left over from the sample, I think

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
Just a couple of smol tweaks, but otherwise this is a really nice leap towards getting the publication page to where it wants to be.

File: cms/publications/templates/publications/publication.html [hunk 2]
```diff
@@ -2,16 +2,79 @@
 {% load wagtailcore_tags %}
 
 {% block content %}
-<article class="nhsuk-width-container">
```
[the remark this answers is not in the document]
Comment (LornaHarwood) [reply cms/publications/templates/publications/publication.html:5]: 👍

File: cms/core/context_processors.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+def request_contexts(request):
+    return {
+        "request_url": request.build_absolute_uri(request.path)
```
[the remark this answers is not in the document]
Comment (LornaHarwood) [reply cms/core/context_processors.py:3]: I'm not getting a url link with the implementation of `{{ publication.full_url }}`
Its coming back blank on the email. 
I'm just getting `publication page:  ` for the body

**Pull request merged**